src/w3c/headers.js

```diff
@@ -731,7 +731,7 @@ async function deriveHistoryURI(conf) {
 
   const canShowHistory = conf.isEd || trStatus.includes(conf.specStatus);
 
-  if (conf.historyURI && canShowHistory) {
+  if (conf.historyURI && !canShowHistory) {
```

Comment:
Good catch! That's what is breaking the broken tests.
The test needs to set `respecConfig.group`, otherwise ReSpec defaults to `specStatus: "base"`, which isn't part of trStatus.

Reply:
Ignore rest of the tests. They're broken due to xref data change.

Reply:
Thanks! Adding the `webapps` group to the options solved the issue.